specs/elements/ActionSet.md

```diff
@@ -45,7 +45,7 @@ Describes what to do when an unknown element is encountered or the requires of t
 * **Allowed values**:
   * `ActionSet`
   * `ColumnSet`
-  * `Container`
```

Comment:
>Container [](http://example.com/codeflow?start=5&length=9)

This is almost certainly not caused by your change, but I'm curious what happened that made Container fall off this list. Also, TableCell and TableRow aren't really fallback options for an ActionSet.

Reply:
yeah - I'm in the middle of tracking some of these issues down :)